lib/api/controllers/securityController.js

```diff
@@ -679,6 +680,27 @@ class SecurityController extends NativeController {
     };
   }
 
+  /**
+   * Given a credentials related search query, returns matched users' kuid.
+   *
+   * @param {Request} request
+   * @returns {Promise<Object>}
+   */
+  async searchUsersByCredentials(request) {
+    const strategy = request.getString('strategy');
+    const query = request.getBodyObject('query');
+
+    this.assertIsStrategyRegistered(strategy);
+
+    const searchMethod = this.getStrategyMethod(strategy, 'search');
+
+    if ( !searchMethod ) {
```

Comment:
Nitpicking: coding standards (we can discuss this)
```suggestion
    if (! searchMethod) {
```